eras/babbage/impl/src/Cardano/Ledger/Babbage/Rules/Utxo.hs

```diff
@@ -440,7 +436,33 @@ utxoTransition = do
 
   {-   ‖collateral tx‖  ≤  maxCollInputs pp   -}
   runTest $ Alonzo.validateTooManyCollateralInputs pp txBody
+  pure utxos
```

Comment:
Might it be better to  return unit perhaps, to make it clear that the function doesn't change the given state?